mnd/mnd/migrations/0017_primary_carer_email_unique_data_migration.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from collections import defaultdict
+import json
+from django.db import migrations, models
+
+
+
+def primary_carer_unique_email(apps, schema_editor):
+    PrimaryCarer = apps.get_model('mnd', 'PrimaryCarer')
+    PrimaryCarerRelationship = apps.get_model('mnd', 'PrimaryCarerRelationShip')
+    Patient = apps.get_model('patients', 'Patient')
+    CustomUser = apps.get_model('groups', 'CustomUser')
+    carer_email_dict = defaultdict(list)
+    for pc in PrimaryCarer.objects.all().order_by('id'):
+        carer_email_dict[pc.email.lower()].append(pc)
+    multiple_carers = { email: carers for email, carers in carer_email_dict.items() if len(carers) > 1}
+    print(f"Found {len(multiple_carers)} email addresses with multiple carers")
```

Comment:
Are these print statements for debugging purposes, or intended to be in the final migration? I would think it's a bad idea to include them.

Reply:
Removed print statements.